tests/processes/BearRunningTest.py

```diff
@@ -253,13 +253,16 @@ def setUp(self):
         self.file_name_queue.put("invalid file")
         self.local_bear_list.append(LocalTestBear(self.settings,
                                                   self.message_queue))
-        self.local_bear_list.append("not a valid bear")
+        self.local_bear_list.append(SimpleGlobalBear({},
+                                                     self.settings,
+                                                     self.message_queue))
```

Comment:
@sils1297 Thanks, I get what you are saying, but this still brings me back to the question I have been asking for more than a week now. Any suggestions on how to handle the fact that a string type can not inherit the `name()` method ?. That is, should we maintain the `bear_instance.__class__.__name__` or add a condition like  
`if not isinstance(bear_instance, Bear)` to handles other types ?.